string id
string type
string time
string mode
string[] errors
r_pose pose
r_pose_optional destination
float64[36] covariance
r_battery battery
